inc/class.client.php

```diff
@@ -324,9 +340,15 @@ public static function output_content( $html_class = '', $format = 'list', $titl
 					$output = $before . '<table class="' . $html_class . ' taxopress-table-container">' . "\n\t";
 					$count = 0;
 					foreach ($content as $item) {
+
+						$term = get_term_by('name', $item, 'post_tag', 'categories', 'link_categories');
```

Comment:
What happen when the term is a custom taxonomy and not in any of the static taxonomy you listed(tags, categories etc)?